packages/dashboard/src/components/pages/trade-type/cfd-test.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import AppToTrade from '../../template/app-to-trade';
+import Instruments from '../../template/instruments';
+import Indices from '../../template/indices';
+import YouMightLike from '../../template/you-might-like';
+import Banner from '../../template/banner';
+
+const card_data = [
+    {
+        title: 'DMT5 Synthetic',
```

Comment:
We should `localize` all the titles and subtitles if it's not temporary ones

Reply:
this page is going to be removed it is just a test page to see my abstract layout

Reply:
Cool, then we don't need translation